Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -11,19 +11,61 @@ static void Main(string[] args)
             Test("42*?7=1974", 4);
             Test("42*?47=1974", -1);
             Test("2*12?=247", -1);
-            Console.ReadKey(true);
+            //Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(String equation){

```

Comment:
You should use 'string' here and not 'String'. 
'string' is an alias provided for 'String' class and it's common practice to use these aliases rather that their actual class names.
You can read more about it here : https://programmingwithmosh.com/net/difference-between-string-and-string-in-c/